docs/API-standaarden.md

```diff
@@ -3,21 +3,37 @@ layout: landing-page
 title: API Standaarden
 ---
 
-# API Standaarden
+# API-standaarden
 
-Het wordt steeds belangrijker voor (applicatie)componenten om onderling goed samen te werken. Om dit mogelijk te maken kennen componenten (applicatie)services die voor andere componenten toegankelijk zijn via zogenaamde 'Application Programming Interfaces' (API's). Zo kunnen componenten op een gestructureerde manier functionaliteit van andere componenten gebruiken. Bijvoorbeeld door een API aan te bieden die voor andere componenten bruikbaar is om gegevens uit een achterliggende bronregistratie te gebruiken.
+Hier vind je de specificaties van de door VNG Realisatie beheerde StUF-standaarden, gericht op ontwikkelaars die met deze standaarden werken.
```

Comment:
Valt me nu pas op dat in deze zin 'StUF-standaarden' staat. Moet dat niet 'API-standaarden' zijn?